app/response_svc.py

```diff
@@ -1,6 +1,26 @@
 from aiohttp_jinja2 import template
+import asyncio
 
 from app.utility.base_service import BaseService
+from app.utility.observer import Observer
+
+
+class LinkObserver(Observer):
+
+    def __init__(self, rest_svc):
+        Observer.__init__(self)
+        self.rest_svc = rest_svc
+
+    def link_added(self):
+        access = dict(access=(self.rest_svc.Access.BLUE,))
+        data = {
+            'name': 'Auto-Collect Blue Data',
```

Comment:
1. use dict() instead of curly braces. 
2. get rid of the hard-coded ability ID. we only do this in 2 other places in the app/plugins-- i don't want to add a 3rd.
3. should group be a string or an "access" object?